app/models/email.rb

```diff
@@ -23,4 +23,11 @@ def self.timed_bulk_insert(records, batch_size)
 
     Metrics.email_bulk_insert(batch_size) { insert_all!(records) }
   end
+
+  def mark_as_spam!
```

Comment:
Although this is a relevant method for `Email`, it's only used once. The trouble with models is that they can contain most methods; in general, we try to locate code elsewhere if possible. Could we keep this logic in the service?

Reply:
Yeah I was in two minds about this. Let's keep the model slim 👌